src/ethereum_test_tools/common/types.py

```diff
@@ -72,6 +72,62 @@ class Storage:
 
     data: Dict[int, int]
 
+    class InvalidType(Exception):
+        """
+        Invalid type used when describing test's expected storage.
+        """
+
+        v: Any
+
+        def __init__(self, v: Any, *args):
+            super().__init__(args)
+            self.v = v
+
+        def __str__(self):
+            """Print exception string"""
+            return f"invalid type for key/value: {self.v}"
+
+    class MissingKey(Exception):
+        """
+        Test expected to find a storage key set but key was missing.
+        """
+
+        k: int
```

Comment:
```suggestion
        key: int
```
seems short enough and improve readability